File: fizzbuzz.py
#!/usr/bin/env python

import sys

def fizzBuzz(number):
    if number % (3 * 5) == 0:
        return "FizzBuzz"
    elif number % 5 == 0:
        return "Buzz"
    elif number % 3 == 0:
        return "Fizz"
    else:
        return number

def main():
    output = list()
    try:
        number = sys.argv[1]
    except IndexError:
        number = 1

    for i in range (1, int(number)):
        value = fizzBuzz(i)
        output.append(value)
    print(output)

if __name__ == "__main__":
    main()

File: fizzbuzz.js
// as it cannot have command line arguments only from 1 - 100
function fizzbuzz(n){
    if (n % (5 *3)) {
        console.log("FizzBuzz");
    }else if (n % 3) {
        console.log("Fizz");
    } else if (n % 5) {
        console.log("Buzz");
    } else {
        console.log(n)
    }
}

for (var i = 1; 1 <= 100; i++) {
    fizzbuzz(i);
}
